### ISSUE TITLE
Consider additional_site_annotations

### ISSUE THREAD

**Issue by skodapetr** [skodapetr/p2rank-funpdbe#7]
We can add rank or score into site ```additional_site_annotations```

**Comment by rdk** [comment]
I would add pocket **score** and (**center_x, center_y, center_z**) from _predictions.csv . 

P2Rank can also transform pocket score to **z-score** (calibrated of true sites only) and **probability** of predicted pocket being a true binding site. I have yet to add them to _predictions.csv .

**Comment by skodapetr** [comment]
The additional information now contain:
```
{
  "score": "2.2181",
  "center": {
    "y": "2.6434",
    "x": "18.6665",
    "z": "21.5826"
  }
}
```

**Comment by skodapetr** [comment]
@rdk as for the z-score you can open new issue once p2rank output is ready for that.
